ner_extended/update_meta_json.py

```diff
@@ -0,0 +1,71 @@
+import json, sys
```

Comment:
Add a docstring

Reply:
Not sure I follow? For the function or for the script?

Reply:
add a docstring to the script, it is something like:

```python
"""
This is a description of what the code in this file does
"""
import json
import sys
```